### PR TITLE
🕵️ Add reusable security settings file

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/settings_ci.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from .settings import *  # noqa
 
-# instead of running tests against temba_test whictest framework creates, run against regular temba database so that
-# other components (i.e. mailroom) can be run against same database
+# instead of running tests against temba_test which the test framework creates, run against regular temba database so
```
Comment (rowanseymour) [inline temba/settings_ci.py:3]: unrelated

**Review by norkans7 — APPROVED** [review]

File: temba/settings_common.py [hunk 2]
```diff
@@ -875,10 +875,7 @@
 AUTHENTICATION_BACKENDS = ("smartmin.backends.CaseInsensitiveBackend",)
 
 AUTH_PASSWORD_VALIDATORS = [
-    {"NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator"},
     {"NAME": "django.contrib.auth.password_validation.MinimumLengthValidator", "OPTIONS": {"min_length": 8}},
```
Comment (rowanseymour) [inline temba/settings_common.py:878]: leaving this in here preserves legacy RapidPro behavior for anyone only using settings_common.py and means unit tests have some validation
